pkg/rpc/response/result/unclaimed_gas.go

```diff
@@ -25,7 +25,7 @@ type unclaimedGas struct {
 func (g UnclaimedGas) MarshalJSON() ([]byte, error) {
 	gas := &unclaimedGas{
 		Address:   address.Uint160ToString(g.Address),
-		Unclaimed: g.Unclaimed.String(),
+		Unclaimed: util.DecimalToString(&g.Unclaimed, 8),
```

Comment:
BTW, it's really a question if `big.Int` is suitable here. Technically, it'd be nice to have GAS values using the same type in every case.